todo/material/task.md

```diff
@@ -0,0 +1,110 @@
+# Aufgabenliste
+
+::: info
+Diese Aufgabe basiert auf _keinem CS50 Practice Problem_.
+
+**Disclaimer:** Diese Aufgabe wurde nicht vom Lehrstuhl herausgegeben und kann Fehler enthalten. Sie dient, wie das gesamte Material von [inf-lab.dev](https://inf-lab.dev), lediglich zu Übungszwecken!
+:::
+
+Wir alle haben verschiedene Aufgaben, jedoch ist es nicht immer einfach sich alle seine Aufgaben zu merken. Daher wollen wir in dieser Aufgabe eine ToDo-Liste entwickeln, welche uns erlaubt unsere Aufgaben über die Konsole zu verwalten.
+
+## Teilaufgabe 1
+
+Wie immer, wenn wir einen vorbereiteten Code bekommen, sollten wir diesen vorerst verstehen bevor wir Änderungen vornehmen können. Lies deshalb den Code in der Datei `todo.py` und beantworte insbesondere folgende Fragen. Wenn du noch etwas unsicher mit der Python Syntax bist, kannst du die vorgegebene Datei `todo.c` zur hilfe nehmen. Diese enthält den selben Code - nur für die Programmiersprache C.
+
+-   Welche Funktionen müssen laut `TODO`-Kommentaren von uns implementiert werden? Was sind ihre Parameter?
+-   Wie genau kann der Benutzer eine Aufgabe hinzufügen, nachdem er das Programm mit `python todo.py` gestartet hat?
+-   Welche Aufgabe haben die `NotImplementedError`s, die bei verschiedenen Funktionen ausgelöst werden?
+
+## Teilaufgabe 2
+
+Nachdem wir nun das Codegerüst verstanden haben, möchten wir in dieser Aufgabe den Grundstein für das Programm legen.
+
+Erstelle daher in der bereitgestellten Datei `todo.py` eine Klasse `ToDo`, welche die eigentliche Aufgabe (`task`) und ob diese abgeschlossen ist (`completed`), speichert.
```

Comment:
Maybe we should give a hint that writing getter and setter functions should be done.

Reply:
I'm not sure what you want with getter's and setters here? We are using simple properties with `self.foo = bar`, we could bloat the task to artificially include those but I dont think that would be useful.